scripts/ci-entrypoint.sh

```diff
@@ -174,6 +183,12 @@ install_addons() {
             copy_secret
         fi
 
+        CCM_CLUSTER_CIDR="${CIDR0}"
```

Comment:
Does it still allow the user to specify `CCM_CLUSTER_CIDR`?

Reply:
No, is there any case where you'd like to be able to set a CCM CIDR that's different from the Cluster pod CIDR range?

Reply:
This occured to me as well: moving away from a random env var value for this seems like an improvement, this should always be the value in the cluster spec (I think?)

Reply:
Got it. No. I think this change is good.